### ISSUE TITLE
Read-Host: Show how to trim leading and trailing whitespace from user input

### ISSUE THREAD

**Issue by mklement0** [MicrosoftDocs/PowerShell-Docs#10088]
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://learn.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/Read-Host?view=powershell-7.3&viewFallbackFrom=powershell-7

### Summary

Typically, any leading or trailing whitespace in user input is considered _incidental_, and the desire is to _remove_ it.

As an aside: The following feature request asks for a `-Trim` switch to support incidental whitespace removal directly.
  * https://github.com/PowerShell/PowerShell/issues/19680

In the absence of direct trimming support in `Read-Host` (at least for now), showing an example of how to trim after the fact would be helpful; e.g.:

```powershell
$userInput = (Read-Host 'Enter something').Trim()
```

**Comment by sdwheeler** [comment]
I don't think this belongs in the `Read-Host` docs for the same reasons cited in https://github.com/PowerShell/PowerShell/issues/19680#issuecomment-1557271553. However, I think a conceptual article that covered string manipulation would be useful and more appropriate.

**Comment by sdwheeler** [comment]
Reopening this issue. We still should create an article about string manipulation.

**Comment by sdwheeler** [comment]
Created #10289 to track new article
